index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+  </head>
```

Comment:
- [ ] **Required:** Please add the `<meta/>` tag viewport in the `<head></head>` of your page, it allows the browser to control the dimensions and scaling of the page.